cloud_formation.yaml

```diff
@@ -201,6 +209,10 @@ Resources:
       HostedZoneId: !Ref route53RecordGroupSetHostedZone
       Name: !Ref lambdaDomainName
       Type: A
+  myLogGroup:
```

Comment:
give this a LogGroupName so we can find it more easily - i suggest something like /cidr/cidrbot